JPF/rent-stab.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import os.path
+import sys
+import logging
+
+
+from sqlalchemy import create_engine
+
+from utils import *
+
+mkdir_p(BASE_DIR)
+
+logging.basicConfig(format='[%(asctime)s] {%(pathname)s:%(lineno)d} %(levelname)s - %(message)s',
+    datefmt='%H:%M:%S',
+    stream=sys.stdout,
+    level=logging.INFO)
+log = logging.getLogger(__name__)
+
+"""
+Taxbills Rent Stabilization joined count import
+"""
+
+TAXBILLS_KEY = 'taxbills_joined'
+
+taxbills_dtype_dict = {
+    "borough": "object",
+    "ucbbl": "int64",
+    "2007uc": "float64", "2007est": "object", "2007dhcr": "object", "2007abat": "object",
+    "2008uc": "float64", "2008est": "object", "2008dhcr": "object", "2008abat": "object",
+    "2009uc": "float64", "2009est": "object", "2009dhcr": "object", "2009abat": "object",
+    "2010uc": "float64", "2010est": "object", "2010dhcr": "object", "2010abat": "object",
+    "2011uc": "float64", "2011est": "object", "2011dhcr": "object", "2011abat": "object",
+    "2012uc": "float64", "2012est": "object", "2012dhcr": "object", "2012abat": "object",
+    "2013uc": "float64", "2013est": "object", "2013dhcr": "object", "2013abat": "object",
+    "2014uc": "float64", "2014est": "object", "2014dhcr": "object", "2014abat": "object",
+    "2015uc": "float64", "2015est": "object", "2015dhcr": "object", "2015abat": "object",
```

Comment:
In a few iterations, maybe...let's get an end-to-end integration working well, first.